CleanWeather/Networking/APIClient.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  APIClient.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 13/12/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+final class APIClient {
+
+    private let defaultSession = URLSession(configuration: .default)
+    private let path = "/api/records/1.0/search/"
+
+    private var dataTask: URLSessionTask?
+    
+    func perform(request: Request) {
+
+        var urlComponents = URLComponents()
+        var allCities = [City]()
+
+        urlComponents.scheme = "https"
+        urlComponents.host = request.url
```

Comment:
query items zrób z request.parameters który ma być typu [string : Any]

Reply:
W którym miejscu ma następować map tego dictionary [String : Any] to [URLQueryItem]?

Reply:
Poradziłem sobie za pomocą URLComponentsExtension, ale nie wiem, czy o to chodziło.